pkg/api/server/register_images.go

```diff
@@ -89,32 +87,41 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	r.Handle(VersionedPath("/images/json"), APIHandler(s.Context, generic.GetImages)).Methods(http.MethodGet)
 	// swagger:operation POST /images/load images loadImage
 	//
-	// Import image
-	//
 	// ---
+	// summary: Import image
+	// description: Load a set of images and tags into a repository.
 	// parameters:
 	//  - in: query
 	//    name: quiet
 	//    type: bool
 	//    description: not supported
+	//  - in: body
+	//    description: tarball of container image
+	//    type: string
+	//    format: binary
```

Comment:
https://docs.docker.com/engine/api/v1.40/#operation/ImageLoad doesn't mention this parameter. Can you elaborate?